docs/hpc/applications/easybuild.md

```diff
@@ -38,195 +198,249 @@ archspec/0.1.4-GCCcore-11.3.0                            SCOTCH/7.0.1-iimpi-2022
+To use the production software stack within a batch job, the `tools/prod` module needs to be used
 
-```console
-[username@login ~]$ gmx
--bash: gmx: command not found
+For example to load GROMACS 2021.3 (with CUDA acceleration) from the production GPU software stack and use within a batch you, you could do the following:
+
+Using production GPU modules within a batch script
+
+```bash
+#PBS -lwalltime=01:00:00
+#PBS -l select=1:ncpus=4:mem=24gb:ngpus=1:gpu_type=RTX6000
+
+ml purge
+ml tools/prod
+ml AlphaFold/2.3.4-foss-2022a-CUDA-11.8.0-ColabFold
+
+<your options>
 ```
 
-### Within a Batch Job
+More information on running GPU jobs may be found on our [GPU Jobs page](../queues/gpu-jobs.md)
 
-To use the production software stack within a batch job, you need to load the tools/prod module rather than the tools/prod-headnode module.
+## User Software Development
+### Using EasyBuild
 
-For example to load GROMACS 2021.5 from the production software stack and use within a batch you, you could do the following:
+In order to make it easier for those users who are either developing software, or need to quickly install some software to test, we are now offering a development module: `tools/eb-dev`.
 
-Using GROMACS 2021.5 from the production software stack in a batch job.
+This allows you to install software with EasyBuild in your own user space, so that you can use a locally modified version of software, or set up a reproducible build environment when working on your own software development projects. The latter will allow you to change your build environment without going through Anaconda or any other virtual environment.
+If you only want to switch compilers etc, you can make use of the already installed `buildenv` modules (see below) which will give you convenient access to various versions of both GCC and Intel compilers, next to MPI access.
 
-```bash
-#PBS -lwalltime=01:00:00
-#PBS -lselect=1:ncpus=16:mem=16gb
+Note: EasyBuild is quite prescriptive, so if you are selecting a specific tool-chain you will get a specific version of the compiler, MPI etc. It is not a mix-and-match approach.
+Note: Remember we got a heterogeneous cluster (CX3-Phase2) regarding the various CPUs we are using (AMD, and Intel). Care needs to be taken when building software from source to make sure it will run on the cluster. Please come to one of our workshops if you are  doing software development seriously.
 
-module purge
-module load tools/prod
-module load GROMACS/2021.5-foss-2021b
+We will be offering hands-on workshops throughout the year to explain the benefits of using EasyBuild for your software development project for example, or simply how to make the best use of the new and exciting software stack. These workshops will be announced via the usual channels.
```

Comment:
You can add message to subscribe to RCS newsletter so that people can get information about the workshops.

Reply:
Very good idea. We should get the link in there as well!

Reply:
That is done now.

Reply:
In general, you should mention commit ID in your comment so that reviewer can see what changes you have made and where. Github links the commit ID and allows to see the change immediately.

For example, you made this change in f8269efbeaea3bc9dfba2acafe1ea2c0317c5fa4.